torchvision/prototype/utils/_internal.py

```diff
@@ -48,8 +48,17 @@
 class StrEnumMeta(enum.EnumMeta):
     auto = enum.auto
 
-    def __getitem__(self, item):
-        return super().__getitem__(item.upper() if isinstance(item, str) else item)
+    def from_str(self, member: str):
+        try:
+            return self[member]
```

Comment:
Does this work with aliases like the `DEFAULT` records of Weights?

Reply:
Confirmed locally and CI seems to be happy too.